app/uk/gov/hmrc/trackingconsentfrontend/views/components/CookieHead.scala.html

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  *@
 
-@this()
+@this(optimizelySnippet: OptimizelySnippet)
 
 @()(implicit appConfig: AppConfig)
 <!--[if !IE]>-->
```

Comment:
I wonder if it would be cleaner if this went in its own template like trackingSnippet?